deploy/docker-swarm/clickhouse-setup/otel-collector-config.yaml

```diff
@@ -1,29 +1,20 @@
 receivers:
-  filelog/dockercontainers:
-    include: [  "/var/lib/docker/containers/*/*.log" ]
-    start_at: end
-    include_file_path: true
-    include_file_name: false
+  tcplog/docker:
+    listen_address: "0.0.0.0:2255"
     operators:
-    - type: json_parser
-      id: parser-docker
-      output: extract_metadata_from_filepath
-      timestamp:
-        parse_from: attributes.time
-        layout: '%Y-%m-%dT%H:%M:%S.%LZ'
-    - type: regex_parser
-      id: extract_metadata_from_filepath
-      regex: '^.*containers/(?P<container_id>[^_]+)/.*log$'
-      parse_from: attributes["log.file.path"]
-      output: parse_body
-    - type: move
-      id: parse_body
-      from: attributes.log
-      to: body
-      output: time
-    - type: remove
-      id: time
-      field: attributes.time
+      - type: regex_parser
+        regex: '^<([0-9]+)>[0-9]+ (?P<timestamp>[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}(\.[0-9]+)?([zZ]|([\+-])([01]\d|2[0-3]):?([0-5]\d)?)?) (?P<container_id>\S+) (?P<container_name>\S+) [0-9]+ - -( (?P<body>.*))?'
+        timestamp:
+          parse_from: attributes.timestamp
+          layout: '%Y-%m-%dT%H:%M:%S.%LZ'
+      - type: move
+        from: attributes["body"]
+        to: body
+      - type: remove
+        field: attributes.timestamp
+        # please add name of containers here if you want to exclude them from logs
+      - type: filter
+        expr: 'attributes.container_name matches "^(:?signoz_)?(logspout|frontend|clickhouse-setup-alertmanager-1|query-service|clickhouse-setup-otel-collector-1|clickhouse-setup-otel-collector-metrics-1|clickhouse|zookeeper-1)'
```

Comment:
Maybe prefix all the signoz container names with `signoz/`. It is certainly possible people might have `clickhouse` or `frontend` in their applications.

Reply:
Have added the change